test/integration/consensus/consensus_sunny_day.cpp

```diff
@@ -15,10 +15,13 @@
  * limitations under the License.
  */
 
+#include <ed25519/ed25519.h>
```

Comment:
May you point out usage of that include?

Reply:
I guess, it redundant to use valid *concrete* implementation in tests.